chapters/robots.adoc

```diff
@@ -19,10 +19,9 @@ The <<Referee, referee>> has to force a team to remove a robot from the field if
 A robot must fit inside a 0.18 meters wide and 0.15 meters high cylinder at any point in time. Additionally, the top of the robot must adhere to the standard pattern size and surface constraints.
 
 ==== Dribbling Device
-Observation: The dribbling device will not be allowed in SSL-EL. Robots can have the device, but it will not be allowed activated during the game.
+The dribbling device, present in SSL division B and A, will not be allowed in SSL-EL. Robots can have the device, but it will not be allowed activated during the game.
```

Comment:
```suggestion
The dribbling device present in the Regular League will not be allowed in the SSL-EL. Robots can have the device, but it will not be allowed to be activated during the game.
```